src/gui/MainWindowAdaptor.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file was generated by qdbusxml2cpp version 0.8
```

Comment:
File should have the usual header, maybe with additional notices.

Reply:
Sincerely, I would feel silly for adding a copyright notice to an autogenerated file

Reply:
Shouldn't this file be generated by the build system then?

Reply:
If we can execute external programs in cmake I would say yes.

`qdbusxml2cpp` comes in `qtbase5-dev-tools` package on debian, so it's another dependency to add on linux.

Probably this file won't be edited for a long time, we should leave this here and don't bother too much, we just need to remember that's autogenerated and the header will come in handy